### DISCUSSION TITLE
Is it possible to run super-linter locally without the Docker container?

### DISCUSSION THREAD

**Discussion by iapazmino** [super-linter/super-linter#5676] [Q&A] [answered]
Hello everyone,

I'd like to run super-linter in my machine without Docker.
There any executable I could call get my code lintered, maybe?
Or any other way to do it without the container?

Thanks,
Ivan

**Comment by ferrarimarco** [comment, marked as answer]
Hi @iapazmino !

To do this, you would need to reproduce what we do when building the container image locally, and it's likely a lot of effort. Example: install dependencies (and keep them up to date), configure environment variables....

I wouldn't advise you to do so. At that point, I would look into installing only the linters you're interested in.
